### PR TITLE
chore: use router bridge from federation-rs

### PR DESCRIPTION
**Pull request by o0Ignition0o**
Iterating on avery's great work moving the rust parts of federation to a separate repository, we can now target the recently public federation-rs repository, which this commit achieves.

### CODE REVIEW

**Review by garypen — APPROVED** [review]
Looks good.

**Review by Geal — APPROVED** [review]

**Comment by abernix** [comment]
Let's land this as soon as https://github.com/apollographql/federation-rs/pull/35 lands and have it point to the resulting commit?  Then we can mark this PR as closing https://github.com/apollographql/router/issues/458, too.

File: apollo-router-core/Cargo.toml [hunk 1]
```diff
@@ -24,10 +24,10 @@ include_dir = "0.7.2"
 lru = "0.7.2"
 miette = { version = "3.3.0", features = ["fancy"] }
 once_cell = "1.9.0"
-router-bridge = { git = "https://github.com/apollographql/federation.git", rev = "950eb931e38746bb7cfed05382d6970a22e43cc4" }
+router-bridge = { git = "https://github.com/apollographql/federation-rs.git", rev = "e72e470fa82f9edd2fc4089e6b30246d50ce5fff" }
```
Comment (abernix) [inline apollo-router-core/Cargo.toml:27]: Let's hold just a second and land this as soon as https://github.com/apollographql/federation-rs/pull/35 lands (and have _this_ point to the resulting commit)?

Then we can mark this PR as closing https://github.com/apollographql/router/issues/458, too.

**Review by abernix — APPROVED** [review]
LGTM, but also we can perhaps hold just a couple hours.

File: apollo-router-core/Cargo.toml (hunk 1 above)
Comment (abernix) [reply apollo-router-core/Cargo.toml:27]: Ok, that just happened!  So the commit would be `6f1a7033d03ff7310644d033508256dcf759a602` (https://github.com/apollographql/federation-rs/commit/6f1a7033d03ff7310644d033508256dcf759a602)

Comment (o0Ignition0o) [reply apollo-router-core/Cargo.toml]: merging if CI passes!

**Review by abernix — APPROVED** [review]

**Pull request merged**